frontend/unit.js

```diff
@@ -0,0 +1,16 @@
+class Unit {
+    constructor(texture, img_size, params) {
+        this.sprite = new PIXI.Sprite(texture);
+        this.params = params;
+
+        this.sprite.anchor.set(0.5, 0.5);
+        this.scale_sprite(this.sprite, img_size);
+    }
+
+    // scale sprite to fit size
+    scale_sprite(sprite, size) {
```

Comment:
you can access to sprite with this.sprite
no need to pass it as argument

Reply:
ok